airbyte-cdk/python/airbyte_cdk/sources/file_based/stream/concurrent/cursor/abstract_concurrent_file_based_cursor.py

```diff
@@ -18,7 +17,7 @@
     from airbyte_cdk.sources.file_based.stream.concurrent.adapters import FileBasedStreamPartition
 
 
-class AbstractConcurrentFileBasedCursor(Cursor, AbstractFileBasedCursor, ABC):
```

Comment:
nit: not something that needs to block this PR, but we should figure out what naming scheme we want in the future. Should this just be an `AbstractFileBasedCursor`?